Data Structures - Trees/Binary Search Tree.py

```diff
@@ -55,6 +55,38 @@ def printt(self,curr_node):
       print(str(curr_node.data))
       self.printt(curr_node.right)
 
```

Comment:
Can you please provide additional Test case screenshots and include the deletion function into BST class?